sdk/core/azure-core/test/ut/azure_core_test.cpp

```diff
@@ -21,7 +16,16 @@ int main(int argc, char** argv)
   signal(SIGPIPE, SIG_IGN);
 #endif
 
-  signal(SIGABRT, AbortHandler);
+  // Declare a signal handler to report unhandled exceptions on Windows - this is not needed for
+  // other OS's as they will print the exception to stderr in their terminate() function.
```

Comment:
nit, indentation not needed:
```suggestion
// Declare a signal handler to report unhandled exceptions on Windows - this is not needed for
// other OS's as they will print the exception to stderr in their terminate() function.
```

Reply:
Indentation is needed because this is inside a function.